site/server/app/com/fortysevendeg/exercises/utils/OAuth2.scala

```diff
@@ -24,7 +30,11 @@ object OAuth2 {
   }
 }
 
-class OAuth2Controller(ws: WSClient, userService: UserServices) extends Controller {
+class OAuth2Controller(
+    implicit
+    transactor: Transactor[Task],
```

Comment:
This is named `T` in other places, we should change all implicit evidences to have a cohesive naming convention.